lib/suspenders/generators/app_generator.rb

```diff
@@ -208,12 +205,12 @@ def setup_gitignore
     end
 
     def setup_bundler_audit
-      say "Setting up bundler-audit"
+      say 'Setting up bundler-audit'
       build :setup_bundler_audit
     end
 
     def setup_spring
-      say "Springifying binstubs"
+      say 'Springifying binstubs'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.